src/pytest_forked/__init__.py

```diff
@@ -68,7 +68,12 @@ def report_process_crash(item, result):
     info = ("%s:%s: running the test CRASHED with signal %d" %
             (path, lineno, result.signal))
     from _pytest import runner
-    call = runner.CallInfo(lambda: 0/0, "???")
+    # pytest >= 4.1
+    has_from_call = getattr(runner.CallInfo, "from_call", None) is not None
```

Comment:
i meant `call = getattr(runner.CallInfo, "from_call", runner.CallInfo)(lambda: 0/0, "???")`

Reply:
yikes! 😁 

TBH I prefer my version because it makes it explicit that there's two code portions involved, one for pytest>=4.1 and the other for pytest<4.1.

Reply:
i an follow that reasoning